src/components/Header/Header.vue

```diff
@@ -12,34 +12,43 @@
 </template>
 
 <script lang="ts">
-import {Vue} from "vue-class-component";
-import {Prop} from 'vue-property-decorator';
-
-export default class Header extends Vue {
-  @Prop({default: ''}) readonly image!: string
-  @Prop({default: ''}) readonly title!: string
-  @Prop({default: ''}) readonly subtitle!: string
-}
+import {defineComponent} from "vue";
+
+const Header = defineComponent({
+  props: {
+    image: {
+      type: String,
+      default: '',
```

Comment:
jak nie przekażesz propa image to będziesz mieć `<img src=' '     ...>` i żadne zdjęcie się nie wyrenderuje. Vue udostępnia opcje `required` która ustawiona na `true` powoduje że ten prop musi być przekazany, inaczej poleci Error. Warto z tego skorzystać, albo w defaulcie dać jakiś prawdziwy link do jakiegoś defaultowego zdjęcia.  https://vuejs.org/v2/guide/components-props.html#Prop-Validation